kong/clustering/init.lua

```diff
@@ -205,14 +205,34 @@ local function fill_empty_hashes(hashes)
   end
 end
 
-function _M:request_version_negotiation()
+
+function _M:call_control_plane(premature)
```

Comment:
The function's name could be simply "version\_negotiation" (though it's already used).

Reply:
no, it does initiate the DP->CP connection.